app/connectors/getPenaltyDetails/HIPPenaltyDetailsConnector.scala

```diff
@@ -34,122 +34,81 @@ import java.util.UUID
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class HIPPenaltyDetailsConnector @Inject() (
-    httpClient: HttpClient,
-    appConfig: AppConfig
-)(implicit ec: ExecutionContext, val config: Configuration)
-    extends FeatureSwitching {
+class HIPPenaltyDetailsConnector @Inject() (httpClient: HttpClient, appConfig: AppConfig)(implicit ec: ExecutionContext, val config: Configuration) extends FeatureSwitching {
 
-implicit val throwingReads: HttpReads[HttpResponse] = new HttpReads[HttpResponse] {
-  override def read(method: String, url: String, response: HttpResponse): HttpResponse = {
-    if (response.status >= 400) throw UpstreamErrorResponse(response.body, response.status)
-    else response
-  }
-}
+  implicit val throwingReads: HttpReads[HttpResponse] = (_: String, _: String, response: HttpResponse) => if (response.status >= 400) throw UpstreamErrorResponse(response.body, response.status) else response
 
-  def getPenaltyDetails(
-      enrolmentKey: AgnosticEnrolmentKey
-  )(implicit hc: HeaderCarrier): Future[HIPPenaltyDetailsResponse] = {
-    val url = appConfig.getHIPPenaltyDetailsUrl(enrolmentKey)
-    val headerCarrier = hc.copy(authorization = None)
-    val headers = buildHeadersV1
+  def getPenaltyDetails(enrolmentKey: AgnosticEnrolmentKey)(implicit hc: HeaderCarrier): Future[HIPPenaltyDetailsResponse] = {
+    val url           = appConfig.getHIPPenaltyDetailsUrl(enrolmentKey)
+    val hcWithoutAuth = hc.copy(authorization = None)
+    val headers       = buildHeadersV1
 
-    logger.info(
-      s"[HIPPenaltiesDetailsConnector][getPenaltyDetails][appConfig.getHIPPenaltyDetailsUrl($enrolmentKey)]- Calling GET $url \nHeaders: $headers"
-    )
+    logger.info(s"[HIPPenaltiesDetailsConnector][getPenaltyDetails][$enrolmentKey]- Calling GET $url \nHeaders: $headers")
 
     httpClient
-      .GET[HIPPenaltyDetailsResponse](
-        url,
-        headers = headers
-      ).map { response =>
-         logger.info(s"[getPenaltyDetails] Successful call to 1812 API")
-         response
+      .GET[HIPPenaltyDetailsResponse](url, headers)(implicitly, hcWithoutAuth, implicitly)
+      .map { response =>
```

Comment:
Can get rid of this because parser is already logging the result
<img width="897" height="92" alt="image" src="https://github.com/user-attachments/assets/3b947334-d957-433f-bb4f-fc406ccb4d56" />